### PR TITLE
[11.0] [ADD] purchase_request_subcontracted_service

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
This module allows to add route purchase request to a service product. 

Depends directly on purchase_request #544

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
Module name should be procurement_service_purchase_request

**Comment by HviorForgeFlow** [comment]
What about purchase_request_procurement_service ? Relating its direct dependency

**Comment by JordiBForgeFlow** [comment]
Ok

File: purchasr_request_procurement_service/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Glue module that allows purchase request on service products.
+
+
+Usage
+=====
+
+Go to product / General Information.
+
+If Product Type is a Services you can choose to Purchase Request.
```
Comment (JordiBForgeFlow) [inline purchasr_request_procurement_service/README.rst]: If Product Type is 'Service', you can select 'Purchase Request'.

File: purchasr_request_procurement_service/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "purchase_request",
+        "product"
+    ],
+    "data": [
+        "views/product_template.xml",
+    ],
+    'demo': [
+    ],
+    "license": 'LGPL-3',
+    'installable': True
```
Comment (JordiBForgeFlow) [inline purchasr_request_procurement_service/__manifest__.py]: add 'auto_install': True,

File: purchasr_request_procurement_service/models/product.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    purchase_request_service = fields.Boolean(
+        string="Purchase Request Service",
+        compute='_compute_purchase_request',
+        inverse='_inverse_purchase_request',
+        )
```
Comment (JordiBForgeFlow) [inline purchasr_request_procurement_service/models/product.py]: Add help="When this option is selected, any procurement for services raising from the inventory subsystem (like services for subcontracting in manufacturing) will translate into a purchase request.

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]

**Comment by JordiBForgeFlow** [comment]
Where is the dependency with 'subcontracted_service'? Name the module 'purchase_request_subcontracted_service'. 

Add tests.

Review Travis.

**Comment by JordiBForgeFlow** [comment]
Perhaps you should add the PR #544 as dependency in https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/oca_dependencies.txt#L15. But not sure if it is allowed, when the PR belongs to the same repo...

**Comment by HviorForgeFlow** [comment]
@jbeficent change already done

File: purchase_request_subcontracted_service/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "depends": [
+        "purchase_request",
+        "product"
+    ],
+    "data": [
+        "views/product_template.xml",
+    ],
+    'demo': [
+    ],
+    "license": 'LGPL-3',
```
Comment (MeetKD) [inline purchase_request_subcontracted_service/__manifest__.py:18]: Can you please add website key and remove empty demo data key ?

**Review by MeetKD — CHANGES_REQUESTED** [review]

**Comment by HviorForgeFlow** [comment]
@jbeficent @MeetKD update review please, Thanks :)

**Comment by HviorForgeFlow** [comment]
@jbeficent @MeetKD Can we move it forward?

**Comment by HviorForgeFlow** [comment]
@jbeficent @MeetKD Can it be moved forward?

**Pull request closed without merging**